### ISSUE TITLE
aws_launch_template: ipv4_address_count cannot be set

### ISSUE THREAD

**Issue by kl4w** [terraform-providers/terraform-provider-aws#5851]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.11.8

* provider.aws: 1.35

### Affected Resource(s)

* aws_launch_template

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
resource "aws_launch_template" "name" {
  network_interfaces {
    ipv4_address_count          = 1
  }
}
```

### Expected Behavior

ipv4_address_count should be able to set

I believe this is due to the field being marked `computed`. After removing that line it seems to be working just fine.

### Actual Behavior

`Error: aws_launch_template.name: "network_interfaces.0.ipv4_address_count": this field cannot be set`

### References

* #5759

**Comment by bflad** [comment]
Support has been merged in and will release with version 1.36.0 of the AWS provider, likely later today. 👍

**Comment by bflad** [comment]
This has been released in [version 1.36.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1360-september-13-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
